hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/SchemaAwareSerializer.java

```diff
@@ -0,0 +1,43 @@
+package pl.allegro.tech.hermes.common.message.wrapper;
+
+import pl.allegro.analytics.commons.avrocodecs.AvroCodecException;
+import pl.allegro.analytics.commons.avrocodecs.internal.AvroSerDe;
+import pl.allegro.tech.hermes.domain.topic.schema.SchemaVersion;
+
+import java.util.Arrays;
+
+public class SchemaAwareSerializer {
```

Comment:
Used quite often is a term "SerDe" for such things, I think for clarity we could also use it here.